compiler/rustc_codegen_cranelift/src/intrinsics/mod.rs

```diff
@@ -1,4 +1,4 @@
-//! Codegen of intrinsics. This includes `extern "rust-intrinsic"`,
+//! Code generation of intrinsics, including functions marked with the `#[rustc_intrinsic]` attribute
```

Comment:
I prefer codegen over code generation.

Reply:
As long as the old `extern "rust-intrinsic"` blocks are still supported, they should be mentioned here. However, it would be good to add `#[rutsc_intrinsic]` to the list.